hardhat.config.ts

```diff
@@ -16,8 +16,5 @@ module.exports = {
                 runs: 200
             }
         }
-    },
-    paths: {
-        artifacts: "./build"
```

Comment:
Now the compiled contracts will go to the `./artifacts` directory. Run `npm run generate` after merging this and can safely remove `./build`

Reply:
Nice!